src/promptflow/promptflow/_sdk/_submitter/run_submitter.py

```diff
@@ -64,7 +66,8 @@ def _run_bulk(self, run: Run, stream=False, **kwargs):
     @classmethod
     def _validate_inputs(cls, run: Run):
         if not run.run and not run.data:
-            raise ValueError("Either run or data must be specified for flow run.")
+            error = ValueError("Either run or data must be specified for flow run.")
+            raise UserErrorException(message=str(error), error=error)
```

Comment:
We might use the ValidationError class here

Reply:
Good catch, I will modify here.

Reply:
The modifications have been completed.